scripts/delayed.js

```diff
@@ -1,26 +1,73 @@
 /* eslint-disable import/no-cycle */
-import { getConfigValue } from './configs.js';
+import { FetchGraphQL } from '@dropins/tools/fetch-graphql.js';
+import { getHeaders } from './configs.js';
 import { getUserTokenCookie } from './initializers/index.js';
 import { getConsent } from './scripts.js';
+import { commerceEndpointWithQueryParams } from './commerce.js';
 
 async function initAnalytics() {
+  const gql = new FetchGraphQL(
+    await commerceEndpointWithQueryParams(),
+    await getHeaders('cs'),
+  );
+
+  // fetch data from storeconfig
+  const { data } = await gql.fetchGraphQl(`
+    query DSContextQuery {  
+      ds: dataServicesStorefrontInstanceContext {
+        storeId: store_id
+        websiteId: website_id
+        environment
+        storeCode: store_code
+        storeViewId: store_view_id
+        baseCurrencyCode: base_currency_code
+        storeViewCurrencyCode: store_view_currency_code
+        storeViewName: store_view_name
+        storeName: store_name
+        websiteName: website_name
+        storeCode: store_code
+        websiteCode: website_code
+        storeUrl: store_url
+        environmentId: environment_id
+        storeViewCode: store_view_code
+      }
+    }
+  `, { method: 'GET' });
+
+  const {
+    storeId,
+    websiteId,
+    environment,
+    storeViewId,
+    baseCurrencyCode,
+    storeViewCurrencyCode,
+    storeViewName,
+    storeName,
+    websiteName,
+    storeCode,
+    websiteCode,
+    storeUrl,
+    environmentId,
+    storeViewCode,
+  } = data?.ds ?? {};
```

Comment:
If response empty, then we should probably warn in console or something.

Reply:
done!